src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -15,7 +17,7 @@
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isYes;
 
-@SuppressWarnings({"PMD.UnnecessaryLocalBeforeReturn","PMD.ReturnEmptyCollectionRatherThanNull"})
+@SuppressWarnings({"PMD.UnnecessaryLocalBeforeReturn","PMD.ReturnEmptyCollectionRatherThanNull", "PMD.GodClass"})
```

Comment:
We shouldn't be adding more suppressions, however I have this branch to fix the hearingLocations method. In it I have extracted the method so we don't have a God class. https://github.com/hmcts/sscs-hearings-api/pull/145/files

Reply:
Noted, no more in the 145 PR